index.d.ts

```diff
@@ -0,0 +1,44 @@
+declare module '@ottofeller/epubjs-rn' {
+  import React from 'react';
+
+  interface Props {
+    style?: object;
```

Comment:
This is style is the react-native style? If positive, consider that changing this from object to real style definition, like `StyleProp<ViewStyle>` for the `View` component.